### PR TITLE
Add new cmd lets

### PR DESCRIPTION
**Pull request by ShaunInIdaho**
- Adding new commands for repo level variables
- Moving environment variable commands from internal
- Adding pipeline configuration commands

Pester tests included

### CODE REVIEW

**Comment by eelkram** [comment]
@ShaunInIdaho while you are working on this can you also try bumping up the version of agent for Mac to xcode9-macos10.15?  The 10.13 version is being retired soon...

File: Atlassian.Bitbucket.Pipeline.Variable.psm1 [hunk 1]
```diff
@@ -0,0 +1,91 @@
+using module .\Atlassian.Bitbucket.Authentication.psm1
```
[the remark this answers is not in the document]
Comment (ShaunInIdaho) [reply Atlassian.Bitbucket.Pipeline.Variable.psm1]: I was trying to follow the outline of the API spec which has variables under pipeline_config.  We call them repo variables but they're actually pipeline config variables according to the API.

File: Atlassian.Bitbucket.Pipeline.psm1 [hunk 2]
```diff
@@ -1,5 +1,56 @@
 using module .\Atlassian.Bitbucket.Authentication.psm1
 
+function Enable-BitbucketPipelineConfig {
```
[the remark this answers is not in the document]
Comment (ShaunInIdaho) [reply Atlassian.Bitbucket.Pipeline.psm1]: Yeah, the only configuration you can do here is enable the pipeline feature for the repo.

[the remark this answers is not in the document]
Comment (eelkram) [reply Atlassian.Bitbucket.Pipeline.psm1]: Ok, i'm fine with that then.

File: Atlassian.Bitbucket.Pipeline.Variable.psm1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (eelkram) [reply Atlassian.Bitbucket.Pipeline.Variable.psm1]: Sounds good

**Comment by eelkram** [comment]
@ShaunInIdaho - the changes proposed look good.  However, you need to update the readme with the updated cmdlets and adjustments for those internal cmdlets that are now public.

**Pull request merged**